project.py

```diff
@@ -81,13 +105,60 @@ def find_board(self, image):
 
         return our_cnt
 
+    def seed_filter(self, image, seed):
+        """
+        My ghetto green filter that only checks if we really hit the board.
+        So it checks if the green value of seed pixel is the highest
+
+        Args:
+            image(numpy.ndarra): Image from witch we get seed values
+            seed(tuple): Seed point
+
+        Returns:
+            bool: True if found the board. False otherwise
+        """
+        rgb = image[self.seed[1], self.seed[0]]
```

Comment:
@Foxiy why does this function have an argument `seed` when it does not seem to be used anywhere?